### PR TITLE
US - accessibility icons

### PR DESCRIPTION
**Pull request by n377i**

### CODE REVIEW

File: lib/useEventForm.js [hunk 1]
```diff
@@ -168,6 +168,24 @@ export function useEventForm(editEvent, categories) {
...
+  // Handle accessibility tag selection
+
+  const [selectedA11yTags, setSelectedA11yTags] = useState([]);
+
+  useEffect(() => {
+    if (editEvent) {
+      setSelectedA11yTags(editEvent.a11yIcons || []);
+    }
+  }, [editEvent]);
+
```
Comment (F-Kirchhoff) [inline lib/useEventForm.js:180]: You don't need this useEffect, simply use the editEvent.a11yIcons as the initialData of your useState: 
```js
  const [selectedA11yTags, setSelectedA11yTags] = useState( editEvent?.a11yIcons || []);
```

Alternatively you can pass a function to useState and its return value will be the initial state:

```js
  const [selectedA11yTags, setSelectedA11yTags] = useState(() => {
    if (editEvent) {
      return editEvent.a11yIcons || [];
    }
    
    return []
  });
```

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
Hi Community Team,

I love what you've done so far. I left some comments for you. It looks like you can take out the useEffects from the EventForm hook, which could help clean things up a bit. Could you check it out? Thanks!

File: lib/useEventForm.js (hunk 1 above)
Comment (n377i) [reply lib/useEventForm.js]: @F-Kirchhoff Thanks! I decided to go for your first suggestion.

**Review by F-Kirchhoff — APPROVED** [review]
LGTM

**Pull request merged**